trollflow2/cli.py

```diff
@@ -0,0 +1,77 @@
+"""Trollflow2 command line interface."""
+
+import argparse
+import json
+import logging
+from datetime import datetime
+from queue import Queue
+
+import yaml
+
+from trollflow2.launcher import logging_on, process_files
+
+logger = logging.getLogger(__name__)
+
+
+def parse_args(args=None):
+    """Parse commandline arguments."""
+    parser = argparse.ArgumentParser(
+        description='Launch trollflow2 processing with Satpy on the provides files then quit.')
+    parser.add_argument("files", nargs='*',
+                        help="Data files to run on",
+                        type=str)
+    parser.add_argument("-p", "--product-list",
+                        help="The yaml file with the product list",
+                        type=str,
+                        required=True)
+    parser.add_argument("-m", "--metadata",
+                        help="Metadata (json) to pass on",
+                        type=str, required=False, default="{}")
+    parser.add_argument("-c", "--log-config",
+                        help="Log config file (yaml) to use",
+                        type=str, required=False, default=None)
+    return parser.parse_args(args)
+
+
+def cli(args=None):
+    """Command line interface."""
+    args = parse_args(args)
+
+    log_config = _read_log_config(args)
+
+    with logging_on(log_config):
```

Comment:
Calling `logging_on(...)` as the outermost call will prevent dask.distributed workers from inheriting log settings, and nothing will be logged (except warnings and errors); see https://github.com/pytroll/trollflow2/issues/209 and https://stackoverflow.com/questions/41475239/how-to-set-up-logging-on-dask-distributed-workers.